genai-perf/README.md

```diff
@@ -617,6 +617,16 @@ An option to enable verbose mode. (default: `False`)
 
 An option to print the version and exit.
 
+##### `--goodput <key:value>`
```

Comment:
This deviates from the other args, like --model and --server-metrics-url are documented. I think this should be <list>.